### DISCUSSION TITLE
Enabling Swtiches

### DISCUSSION THREAD

**Discussion by g-off-white** [travisghansen/hass-pfsense#62] [Q&A] [answered]
Love your work with the integration. However I seem to be missing something.  I want to turn on and off firewall rules.  How do you enable and configure the switches?

**Comment by travisghansen** [comment, marked as answer]
They are disabled by default. View the disabled entities and selectively enable the specific rules you wish to toggle.

**Reply by g-off-white** [reply]
Before I thought I was being thick.  Now I know. Thanks :)

**Reply by travisghansen** [reply]
Enjoy!
